aspnetcore/aspnetcore-2.0.md

```diff
@@ -137,14 +121,12 @@ For information about the status of planned documentation, see the [GitHub issue
 
 ASP.NET Core has always helped HTMLEncode your content by default, but with the new version we’re taking an extra step to help prevent cross-site request forgery (XSRF) attacks: ASP.NET Core will now emit anti-forgery tokens by default and validate them on form POST actions and pages without extra configuration.
 
-For information about the status of planned documentation, see the [GitHub issue](https://github.com/aspnet/Docs/issues/3688).
+For more information, see [Preventing Cross-Site Request Forgery (XSRF/CSRF) Attacks in ASP.NET Core](xref:security/anti-request-forgery).
 
 ## Automatic precompilation
 
 Razor view pre-compilation is enabled during publish by default, reducing the publish output size and application startup time.
 
-For information about the status of planned documentation, see the [GitHub issue](https://github.com/aspnet/Docs/issues/3547).
-
 ## Razor support for C# 7.1
 
 The Razor engine has been updated to work with the new Roslyn compiler. That includes support for C# 7.1 features like Default Expressions, Inferred Tuple Names, and Pattern-Matching with Generics.  To use C #7.1 in your project, add the following property in your project file and then reload the solution:
```

Comment:
- Razor engine --> Razor view engine
- Remove extra space before "To use"
- C #7.1 --> C# 7.1